test/Controllers/StatusInfoControllerTest.scala

```diff
@@ -48,10 +46,6 @@ class StatusInfoControllerTest extends UnitSpec with OneServerPerSuite with Mock
 
   "For API 11, Status Info Controller " should {
 
-    "use the correct status info service" in {
-      StatusInfoController.statusInfoService shouldBe EtmpStatusInfoService
```

Comment:
why removed?

Reply:
I recommended removing this as its a test check the object being used is the correct implementation for that trait. However, this was causing complications and is an unnecessary test.

Reply:
done